qcodes/instrument_drivers/Keysight/N52xx.py

```diff
@@ -73,6 +77,7 @@ def get_raw(self) -> Sequence[float]:
 
         return data
 
+
```

Comment:
I'm going to add comments on the example notebook here:
* please go through the text and fix the typos :)
* you can use `load_or_create_experiment` instead of `try-catch` when an experiment is being loaded
* why haven't you created a file for `N5222B` model? just extract it from the notebook
* first, IP address is mentioned, but then GPIB is used. perhaps, clarify that part so that there is no confusion
* in the 2nd example, you register `phase` and then 'get' `unwrapped_phase`. is it correct? does it need to be clarified in the text of the notebook as well?
* what is `try:     while True:         pass` at the end of the notebook for? :)